wafamole/payloadfuzzer/sqlfuzzer.py

```diff
@@ -191,24 +204,33 @@ def swap_keywords(payload):
 
     symbols = {
         # OR
-        "||": [" OR ", " || "],
-        " || ": [" OR ", "||"],
-        "OR": [" OR ", "||"],
-        "  OR  ": [" OR ", "||", " || "],
+        "||": [" OR ", "OR", " || ", "or", " or "],
```

Comment:
Hello and thanks.
Indeed, you are right. The space is required for operators such as `LIKE` but is not needed for others such as `=`.
I just pushed a new commit that tries to simplify the symbols dict to address the case when there is or not the space.
I also added some unit tests in my [improve_manipulations](https://github.com/biagiom/WAF-A-MoLE/commit/107bd83b0b8842a11ae0a666e876d9201e0946b8) branch.
Since all the manipulations are based on random, I created the corresponding non-random variant in `manipulations_static.py` to perform the tests.

Reply:
Thanks, merged

> I also added some unit tests in my [improve_manipulations](https://github.com/biagiom/WAF-A-MoLE/commit/107bd83b0b8842a11ae0a666e876d9201e0946b8) branch.
That's great :muscle: please open another PR for that